include/exadg/structure/material/library/st_venant_kirchhoff.h

```diff
@@ -69,16 +69,28 @@ class StVenantKirchhoff : public Material<dim, Number>
                     StVenantKirchhoffData<dim> const &      data);
 
   dealii::Tensor<2, dim, dealii::VectorizedArray<Number>>
-  evaluate_stress(dealii::Tensor<2, dim, dealii::VectorizedArray<Number>> const & E,
-                  unsigned int const                                              cell,
-                  unsigned int const                                              q) const final;
+  second_piola_kirchhoff_stress(
+    dealii::Tensor<2, dim, dealii::VectorizedArray<Number>> const & strain_measure,
+    unsigned int const                                              cell,
+    unsigned int const                                              q) const final;
 
   dealii::Tensor<2, dim, dealii::VectorizedArray<Number>>
-  apply_C(dealii::Tensor<2, dim, dealii::VectorizedArray<Number>> const & E,
-          unsigned int const                                              cell,
-          unsigned int const                                              q) const final;
+  second_piola_kirchhoff_stress_derivative(
+    dealii::Tensor<2, dim, dealii::VectorizedArray<Number>> const & Grad_delta,
+    dealii::Tensor<2, dim, dealii::VectorizedArray<Number>> const & F_lin,
```

Comment:
I am sure if this change should be made in the present PR.

The names are somewhat cryptic. Better would be "gradient_of_displacement_increment" and "deformation_gradient_in_linearization_point".